libbeat/template/load_integration_test.go

```diff
@@ -237,7 +237,7 @@ func TestOverwrite(t *testing.T) {
 		Enabled: true,
 		Fields:  absPath + "/fields.yml",
 	})
-	loader, err := NewLoader(config, client, beatInfo)
+	loader, err := NewLoader(config, client, beatInfo, []byte{})
```

Comment:
Can you pass `nil` instead of allocating an empty slice?

Reply:
changed